src/defs/client.ts

```diff
@@ -1779,15 +1770,6 @@ tmpDef.argDescs = {};
 tmpDef.scriptSide = ScriptSide.Client;
 ClientDefinitions.push(tmpDef);
 
-tmpDef = new MTAFunction;
-tmpDef.label = "setPlayerHudComponentVisible";
```

Comment:
This seems to be missing now in your client.ts? setPlayerHudComponentVisible is shared BUT it has different definitions for client / server side.